Cython/Compiler/ExprNodes.py

```diff
@@ -14419,9 +14431,23 @@ def __init__(self, pos, expr, string=None):
             string = StringNode(pos, unicode_value=string, value=string.as_utf8_string())
         self.string = string
         self.expr = expr
+        self.string_annotation = string_annotation
```

Comment:
These should rather be named `is_string_annotation`. They are not referring to the actual (string) annotations.